packages/hint/src/lib/enums/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from '@hint/utils/dist/src/types/category';
 export * from './error-status';
 export * from './hint-scope';
-export * from './resource-type';
+export * from '@hint/utils/dist/src/types/resource-type';
```

Comment:
Similarly, shouldn't this still just export from `./resource-type` as it already re-exports the necessary types?
```suggestion
export * from './resource-type';
```